lib/src/rules/use_super_initializers.dart

```diff
@@ -72,46 +72,54 @@ class _Visitor extends SimpleAstVisitor {
 
   _Visitor(this.rule, this.context);
 
-  void check(SuperConstructorInvocation superInvocation,
+  void check(
+      ConstructorDeclaration node,
+      SuperConstructorInvocation superInvocation,
       FormalParameterList parameters) {
     var constructorElement = superInvocation.staticElement;
     if (constructorElement == null) return;
 
     // todo(pq): consolidate logic shared w/ server
     //  (https://github.com/dart-lang/linter/issues/3263)
 
+    var identifiers = _checkForConvertiblePositionalParams(
+        constructorElement, superInvocation, parameters);
+
     // Bail if there are positional params that can't be converted.
-    if (!_checkForConvertiblePositionalParams(
-        constructorElement, superInvocation, parameters)) {
-      return;
-    }
+    if (identifiers == null) return;
```

Comment:
I'll take another pass with some improved testing. It's likely I'm being too conservative here (and possibly elsewhere).  Thanks!

Reply:
Should there be any difference between the cases the quick assist  for super initializers can work and the cases this lint fires on? If not, it would be nice to test on exactly the same examples.

Reply:
As far as I can see there shouldn't be. Today, however, there are because I haven't finished supporting positional parameters in the fix. (Work is underway, but not complete.) Once that's done and we've verified that every linted case is covered by the fix, we can look at making them the same.